lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -370,6 +383,18 @@ class BasicBlockCloner : public SILClonerWithScopes<BasicBlockCloner> {
       return bbIt->second;
     return bb;
   }
+
+  SILValue remapValue(SILValue Value) {
+    auto VI = ValueMap.find(Value);
+    if (VI != ValueMap.end())
+      return VI->second;
+    return Value;
+  }
+
+  void updateValueMap(SILValue oldValue, SILValue newValue)  {
+    auto emplaceResult = ValueMap.try_emplace(oldValue, newValue);
+    assert(emplaceResult.second && "Remapping value multiple times during SESE cloning.");
```

Comment:
Nit: The message is inconsistent with the semantics here. "Remap value" verbatim refers to the `::remapValue` method above, but that function _can_ be called multiple times. How about changing the message to something along the lines of "inserting the same mapping twice"?